packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -167,6 +168,7 @@ const Autocomplete = <
         hasVisibleLabel
         id={InputLabelProps.htmlFor}
         hint={hint}
+        hintLinkComponent={hintLinkComponent}
```

Comment:
The convention I've seen in MUI is that they use TitleCasing for props that are components. I believe it would be better to call this one `HintLinkComponent` to keep with the convention.

Reply:
I'm fine with that. If that is a standard convention we'd like to start following. @KevinGhadyani-Okta @jordankoschei-okta Any thoughts on this?